cdc/sorter/unified/unified_sorter.go

```diff
@@ -114,7 +114,7 @@ func (s *Sorter) Run(ctx context.Context) error {
 
 	defer close(s.closeCh)
 
-	finish := util.MonitorCancelLatency(ctx, "Unified Sorter")
+	startCancel, finish := util.MonitorCancelLatency(ctx, "Unified Sorter")
```

Comment:
```suggestion
	finish, startCancel := util.MonitorCancelLatency(ctx, "Unified Sorter")
```

Putting cancel in seconds position is more idiomatic.

Reply:
done